scripts/build/bootstrap/incremental_build_util.py

```diff
@@ -320,10 +320,14 @@ def mount_volume_to_device(created):
         time.sleep(1)
 
     else:
-        subprocess.call(['file', '-s', '/dev/xvdf'])
+        device_name = '/dev/xvdf'
+        nvme_device_name = '/dev/nvme1n1'
+        if os.path.exists(nvme_device_name):
+            device_name = nvme_device_name
+        subprocess.call(['file', '-s', device_name])
         if created:
-            subprocess.call(['mkfs', '-t', 'ext4', '/dev/xvdf'])
-        subprocess.call(['mount', '/dev/xvdf', MOUNT_PATH])
+            subprocess.call(['mkfs', '-t', 'ext4', device_name])
+        subprocess.call(['mount', device_name, MOUNT_PATH])
```

Comment:
doesnt line 335 have to be updated as well, I am also seeing mention to xvdf other times
`delete_branch_ebs.py` script is also mentioning that device

Reply:
It doesn't need to be updated there since that's just the device name provided to EBS. It will then be converted to the required device name in Linux or drive letter in Windows. 

I'll take a look at delete_branch_ebs.py and our other maintenance scripts as well.

Reply:
I will then mark this as approved, leave it up to you to figure if it is needed to update those other scripts at the same time

Reply:
Sounds good. I'm reviewing them now, at a quick glance doesn't look like it will affect the cleanup scripts since this update only affects how it mounts the local device. But I'll double check before I merge it in.